assets/js/pages/SettingsPage/SettingsPage.jsx

```diff
@@ -287,6 +301,8 @@ function SettingsPage() {
               onRetry={() => dispatch(fetchSoftwareUpdatesSettings())}
             >
               <SuseManagerConfig
+                userAbilities={abilities}
+                configEditPermittedFor={sumaSettingsPermittedFor}
```

Comment:
Do you need to pass the `configEditPermittedFor` to SUMA (the same for activility log`)?
It looks something internal for that component.
You could create the constant there, and avoid this extra parameter, which in reality doesn't add any value

Reply:
No you are fully right, it is much cleaner to have a const in the own component